### PR TITLE
Sort by lable

### PR DESCRIPTION
**Pull request by shawn320**
# Description

Display minority list on the Signup page in alpabetical order for both English and French. 

- Added a sort function in ```OptionalListField``` component (credit to @var-kyle )
- fix spacing between options

### CODE REVIEW

File: components/molecules/OptionalListField.js [hunk 1]
```diff
@@ -147,4 +156,9 @@ OptionalListField.propTypes = {
    * legend text for the list items
    */
   listLabel: PropTypes.string,
+
+  /**
+   * sort the children being passed in based on labels
+   */
+  sortedChildren: PropTypes.func,
```
Comment (var-kyle) [inline components/molecules/OptionalListField.js:163]: You don't need to add this since `sortedChildren` is not being passed into the component.

**Review by var-kyle — CHANGES_REQUESTED** [review]
Looks great, just one little change.

**Review by will0684 — APPROVED** [review]
Looks and works great on desktop and mobile great work @shawn320 and @var-kyle

**Review by var-kyle — APPROVED** [review]
Awesome, great job!

**Pull request merged**